src/status_im/ui/screens/chat/components/reply.cljs

```diff
@@ -81,15 +81,16 @@
        [icons/icon :main-icons/close-circle {:container-style (styles/close-button)
                                              :color (:icon-02 @quo.colors/theme)}]]]]))
 
-(defn reply-message [{:keys [from identicon content-type contentType parsed-text content]} in-chat-input?]
+; This component is also used for quoted pinned message as the UI is very similar
+(defn reply-message [{:keys [from identicon content-type contentType parsed-text content]} in-chat-input? pin?]
   (let [contact-name       @(re-frame/subscribe [:contacts/contact-name-by-identity from])
         current-public-key @(re-frame/subscribe [:multiaccount/public-key])
         content-type       (or content-type contentType)]
-    [rn/view {:style {:flex-direction :row :height 24}}
-     [rn/view {:style (styles/reply-content)}
-      [icons/icon :main-icons/connector {:color (theme-colors quo2.colors/neutral-40 quo2.colors/neutral-60)
-                                         :container-style {:position :absolute :left 10 :bottom -4 :width 16 :height 16}}]
-      [rn/view {:style {:position :absolute :left 34 :top 3 :flex-direction :row :align-items :center :width "45%"}}
+    [rn/view {:style {:flex-direction :row :height (when-not pin? 24)}}
+     [rn/view {:style (styles/reply-content pin?)}
```

Comment:
consider using explicit variable names, character saving like this adds little value in my opinion.
.ie `pinned?` in place of `pin?`

Reply:
So here `pin?` like `isPin`. I am using it as a noun rather than a verb to refer to the new system message showing that a user has pinned a message - not the pinned message itself.

Reply:
ok, personally I would use `is-pin?` here so.

Reply:
Same, but I believe it is a Clojure convention to replace `is-` with a `?` at the end.